app/jobs/database_migrations/populate_add_ons_taxes_with_organization_job.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module DatabaseMigrations
+  class PopulateAddOnsTaxesWithOrganizationJob < ApplicationJob
+    queue_as :low_priority
+    unique :until_executed
+
+    BATCH_SIZE = 1000
+
+    def perform(batch_number = 1)
+      batch = AddOn::AppliedTax.unscoped
+        .where(organization_id: nil)
+        .limit(BATCH_SIZE)
+
+      if batch.exists?
+        # rubocop:disable Rails/SkipsModelValidations
+        batch.update_all("organization_id = (SELECT organization_id FROM add_ons WHERE add_ons.id = add_ons_taxes.add_on_id)")
+        # rubocop:enable Rails/SkipsModelValidations
+
+        # Queue the next batch
+        self.class.perform_later(batch_number + 1)
+      else
+        Rails.logger.info("Finished the execution")
```

Comment:
Kind of unrelated to your PR but seeing this, I'm assuming that the job name is already attached to the logs.

I'm [adding a log here](https://github.com/getlago/lago-api/pull/3535/files#diff-3f25050769ae05d8823748244efb284d1b17b6d5a3f0cb76f55a0345b5844a87R13) and I think I'm doing it wrong then.